.circleci/config.yml

```diff
@@ -84,95 +105,38 @@ jobs:
             - ./node_modules
             - ~/.cache # Cypress stores its stuff here.
       - persist_to_workspace:
-          root: *working_dir
+          root: << pipeline.parameters.working_directory >>
           paths:
             - ./*
 
-  lint:
-    executor: base-executor
-    working_directory: *working_dir
-    steps:
-      - attach_workspace: *attach_options
-      - run: yarn lint
-
-  test_sdk_unit:
-    executor: base-executor
-    working_directory: *working_dir
-    steps:
-      - attach_workspace: *attach_options
-      - run:
-          name: Run unit tests
-          command: yarn workspace raiden-ts run test:unit --ci --runInBand --reporters=default --reporters=jest-junit
-          environment:
-            JEST_JUNIT_OUTPUT_DIR: "reports/junit"
-      - store_test_results:
-          path: reports/junit
-      - store_artifacts:
-          path: reports/junit
-      - run:
-          name: "Upload coverage"
-          command: bash <(curl -s https://codecov.io/bash) -F sdk_unit -F sdk -C $CIRCLE_SHA1
-
-  test_sdk_integration:
-    executor: base-executor
-    working_directory: *working_dir
-    steps:
-      - attach_workspace: *attach_options
-      - run:
-          name: Run unit tests
-          command: yarn workspace raiden-ts run test:integration --ci --runInBand --reporters=default --reporters=jest-junit
-          environment:
-            JEST_JUNIT_OUTPUT_DIR: "reports/junit"
-      - store_test_results:
-          path: reports/junit
-      - store_artifacts:
-          path: reports/junit
-      - run:
-          name: "Upload coverage"
-          command: bash <(curl -s https://codecov.io/bash) -F sdk_integration -F sdk -C $CIRCLE_SHA1
-
-  test_dapp_unit:
+  check_licenses:
     executor: base-executor
-    working_directory: *working_dir
+    working_directory: ~/
     steps:
       - attach_workspace: *attach_options
+      # TODO: Export installation into Dockerfile?
       - run:
-          name: Run unit tests
-          command: yarn workspace raiden-dapp run test:unit --ci --runInBand --reporters=default --reporters=jest-junit
-          environment:
-            JEST_JUNIT_OUTPUT_DIR: "reports/junit"
-      - store_test_results:
-          path: reports/junit
-      - store_artifacts:
-          path: reports/junit
-      - run:
-          name: "Upload coverage"
-          command: bash <(curl -s https://codecov.io/bash) -F dapp -C $CIRCLE_SHA1
-
-  test_dapp_e2e:
-    executor: e2e-environment-executor
-    working_directory: *working_dir
-    steps:
-      - attach_workspace: *attach_options
+          name: Update $PATH
+          command: echo 'export PATH=$PATH:$HOME/node_modules/.bin' >> $BASH_ENV
```

Comment:
Meddling with `PATH` isn't pretty.. since we have `yarn` available, maybe it'd be worth it to use `yarn exec`? I think maybe it doesn't even require installing explicitly, and do it automatically

Reply:
Let me try. This might be a cool change. At least my comment above becomes regardless since we don't have a custom Docker image anymore.

Reply:
Btw: in the package.json we have a `npx` command to allow only yarn. So `npx` is apparently available. Why not simply using it here as well?

Reply:
Yes, you can as well. npx should be available wherever npm is, so it's safe to assume it'd be there, and it also can auto-download libs as needed

Reply:
I tested it locally with their Docker image and it looks fine. So I'll change that in the config and we should be fine.

Reply:
:heavy_check_mark: